corehq/apps/hqadmin/urls.py

```diff
@@ -64,6 +65,7 @@
     url(r'^superuser_management/$', SuperuserManagement.as_view(), name=SuperuserManagement.urlname),
     url(r'^phone/restore/$', AdminRestoreView.as_view(), name="admin_restore"),
     url(r'^phone/restore/(?P<app_id>[\w-]+)/$', AdminRestoreView.as_view(), name='app_aware_admin_restore'),
+    url(r'^app_build_timings/$', AppBuildTimingsView.as_view(), name="app_build_timings"),
```

Comment:
If this view were in app manager, you could then make the url something like `/a/DOMAIN/apps/timing/APP_ID/`, which might be easier to remember than `hq/admin/app_build_timings/?app_id=APP_ID` and you could also put a link in app manager to the current app's timings. I'm opposed to magic URLs without links anywhere in the UI. On the other hand, if you're the sole user of this page, I don't think it's that big a deal. Conceptually I think you can make an argument for the view going either in app manager or in hqadmin.

Reply:
+1 for something similar to apps/timings/APP_ID or apps/view/APP_ID/timing

Reply:
Do you think superuser-only views belong in app manager?  Doesn't look like there are any in there right now.  It would be handy to make it domain-specific, I suppose.

I'd like to start this off as a hidden page, then as it proves useful, add a link somewhere, maybe a widget to select an app, and so forth.

Reply:
The import app page is effectively superuser-only, though I forget whether that's enforced or just de facto because there's no link to it. A better example is the domain app, which contains domain-specific superuser views like Calculated Properties. I think of hqadmin as a junk drawer for admin functionality that doesn't fit anywhere else.

I understand not wanting to put the time into an app-selecting widget, but a link is almost no effort. Since it's superuser-only it's not a huge deal where it goes - doing a clock/hourglass icon link next to the Make New Version button could be cute.

Reply:
I do like the idea of a link from the releases page, but I don't think this is quite there yet.  The current intended audience is me and maybe a couple others on Solutions.  I wouldn't want it to show up for GS, for instance.  It could go behind the SUPPORT feature flag.  Still I'd prefer to wait until a later pass to put this somewhere where people will happen across it.